src/components/Task/TaskRow/TaskRow.tsx

```diff
@@ -50,14 +54,54 @@ const TaskType = styled(Typography)(({ theme }) => ({
   marginRight: theme.spacing(0.5),
 }));
 
-const TaskContactName = styled(ContactText)(({ theme }) => ({
-  fontWeight: 700,
-  whiteSpace: 'nowrap',
-  marginRight: theme.spacing(0.5),
-  '&:hover': {
-    textDecoration: 'underline',
-  },
-}));
+type OnClickFunction = (
+  event: React.MouseEvent<HTMLElement, MouseEvent>,
+  contactId: string,
+) => void;
+
+type TaskRowContactNameProps = {
+  contact: TaskRowFragment['contacts']['nodes'][0];
+  itemIndex: number;
+  contactsLength: number;
+  selectContact: OnClickFunction;
+} & TypographyProps;
+
+const TaskRowContactName: React.FC<TaskRowContactNameProps> = ({
```

Comment:
This component is getting long enough to where I'd consider putting it in a separate file.

Reply:
Done